### PR TITLE
Feature/render a table

### PR DESCRIPTION
**Pull request by m0ns3**
In this PR, we were working on render a Sider with the entities that we received from Simfinity API, and a Table is shown with the data, when each entity is clicked. Table not contain filters yet.

### CODE REVIEW

File: src/direflow-components/Table/Table.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { Table as TableAntd } from "antd";
+import { requestEntity } from "./utils";
+import { capitalize } from "../../utils/utils_string";
+
+const Table = (props) => {
```
Comment (ldistasi) [inline src/direflow-components/Table/Table.js:6]: a lo mejor podemos destructurar. Creo que quedaria algo asi:

{ displayEntities }

File: src/direflow-components/Table/utils.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+export const requestEntity = async (displayEntities) => {
+  const entityName = displayEntities.queryAll;
+  const fields = displayEntities.fields;
+  let queryFields = [];
+  for (let i = 0; i < fields.length; i++) {
```
Comment (ldistasi) [inline src/direflow-components/Table/utils.js:7]: Quizas un **_for of_** aca tambien?

File: src/direflow-components/Table/utils.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    const config = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      url: "https://multiscreen-techgroup.rj.r.appspot.com/graphql",
+      data,
+    };
+
+    const response = await axios(config);
+    const responseData = response.data && response.data.data;
+    return responseData;
```
Comment (ldistasi) [inline src/direflow-components/Table/utils.js:45]: creo que se puede simplificar a `return response.data && response.data.data;`

File: src/direflow-components/App.js [hunk 4]
```diff
@@ -1,63 +1,96 @@
...
   sampleList: [
-    'Create with React',
-    'Build as Web Component',
-    'Use it anywhere!',
+    "Create with React",
+    "Build as Web Component",
+    "Use it anywhere!",
   ],
-}
+};
```
Comment (emiacc) [inline src/direflow-components/App.js:93]: I think we can remove 'componentTitle' and 'sampleList' from defaultProps and from propTypes below

File: src/direflow-components/App.js [hunk 5]
```diff
@@ -1,63 +1,96 @@
...
 
-const EntitiesContext = React.createContext();
+const { Title, Paragraph } = Typography;
 
 const App = (props) => {
-  const [entities, setEntities] = useState();
+  const { Header, Content, Footer, Sider } = Layout;
+  const [collapsed, setCollapsed] = useState(false);
+
+  const EntitiesContext = React.createContext();
```
Comment (emiacc) [inline src/direflow-components/App.js:15]: It's better to move this declaration outside of the component

File: src/direflow-components/App.js [hunk 6]
```diff
@@ -1,63 +1,96 @@
...
+import { requestEntities } from "./utils";
+import Table from "./Table/Table";
+import styles from "./App.css";
 
-const EntitiesContext = React.createContext();
+const { Title, Paragraph } = Typography;
 
 const App = (props) => {
-  const [entities, setEntities] = useState();
+  const { Header, Content, Footer, Sider } = Layout;
```
Comment (emiacc) [inline src/direflow-components/App.js:12]: It's better to move this declaration outside of the component

File: src/direflow-components/Table/Table.css [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  background-color: #ddd;
+}
+
+#students th {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: center;
+  background-color: #4caf50;
+  color: white;
+}
```
Comment (emiacc) [inline src/direflow-components/Table/Table.css:34]: Do we need this file?

File: src/direflow-components/Table/Table.js (hunk 1 above)
Comment (emiacc) [reply src/direflow-components/Table/Table.js:6]: I agree, also we can define a propType object for this component

File: src/direflow-components/Table/Table.js [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+  const [filteredInfo, setFilteredInfo] = useState(null);
+  const [sortedInfo, setSortedInfo] = useState(null);
+  const [currentEntity, setCurrentEntity] = useState([]);
+  const [columns, setColumns] = useState([]);
+  const [keys, setKeys] = useState([]);
+
+  useEffect(() => {
+    if (props.displayEntities) {
+      setColumns(
+        props.displayEntities.fields.map((entity) => {
```
Comment (emiacc) [inline src/direflow-components/Table/Table.js:16]: I think we can improve readability a little bit here

**Review by joelmolina2121 — APPROVED** [review]

**Pull request merged**